depthai_examples/src/stereo_inertial_publisher.cpp

```diff
@@ -0,0 +1,277 @@
+                                                                                        &rgbConverter,
+                                                                                        std::placeholders::_1, 
+                                                                                        std::placeholders::_2) , 
+                                                                                        30,
+                                                                                        rgbCameraInfo,
+                                                                                        "color");
+            rgbPublish.addPubisherCallback();
+            ros::spin();
+        }
+        else {
+            dai::rosBridge::BridgePublisher<sensor_msgs::Image, dai::ImgFrame> leftPublish(leftQueue,
+                                                                                            pnh, 
+                                                                                            leftPubName,
+                                                                                            std::bind(&dai::rosBridge::ImageConverter::toRosMsg, 
+                                                                                            &converter, 
+                                                                                            std::placeholders::_1, 
+                                                                                            std::placeholders::_2) , 
+                                                                                            30,
+                                                                                            leftCameraInfo,
+                                                                                            "left");
+            dai::rosBridge::BridgePublisher<sensor_msgs::Image, dai::ImgFrame> rightPublish(rightQueue,
+                                                                                            pnh, 
+                                                                                            rightPubName,
+                                                                                            std::bind(&dai::rosBridge::ImageConverter::toRosMsg, 
+                                                                                            &rightconverter, 
+                                                                                            std::placeholders::_1, 
+                                                                                            std::placeholders::_2) , 
+                                                                                            30,
+                                                                                            rightCameraInfo,
+                                                                                            "right");  
```

Comment:
if we are not publishing left and right when aligned to rgb. We can remove the xlink of these in pipeline init to avoid unused usb bandwidth.